### DISCUSSION TITLE
Inquiry on how to modify kimai css

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4925] [Plugins]
![스크린샷 2024-06-21 144914](https://github.com/kimai/kimai/assets/165639563/20b08ac5-4679-4167-b92d-2997e8fdbd07)
![화면 캡처 2024-06-21 144727](https://github.com/kimai/kimai/assets/165639563/d74ecce6-e88c-4878-bea1-77f56e4170e4)

I would like to adjust the height of the explanation column in the pop-up window for creating customers, projects, activities, and timesheets in Kimai.

I'd like to add the height attribute to the text area below and adjust the height.

Can you tell me how to modify the css file on kimai/public/build/?

**Comment by kevinpapst** [comment]
You can use a 
- symfony form extension
- custom css bundle
- recompile assets (not recommended)

**Reply by seleondevel** [reply]
@kevinpapst I solved it easily with custom css bundle, thank you.

**Reply by Rudiberto** [reply]
>  I solved it easily with custom css bundle, thank you.

Could you be so kind to share your experience with a little bit more details? Amplifying the description field is something that I would also like to do.

**Reply by kevinpapst** [reply]
https://www.kimai.org/store/keleo-css-custom-bundle.html and then follow the link to the documentation for examples
